ecs-agent/metrics/constants.go

```diff
@@ -26,4 +26,9 @@ const (
 	// AttachResourceResponder
 	attachResourceResponderNamespace = "ResourceAttachment"
 	ResourceValidationMetricName     = attachResourceResponderNamespace + ".Validation"
+
+	// TaskManifestResponder, TaskStopVerificationACKResponder
+	taskManifestResponderNamespace = "TaskManifestResponder"
+	TaskStoppedMetricName          = taskManifestResponderNamespace + ".TaskStopped"
```

Comment:
Should we have separate metrics between TaskManifestResponder and TaskStopVeificationACKResponder, since they are separated into two different responders? i.e.,

```
// TaskManifestResponder
	taskManifestResponderNamespace = "TaskManifestResponder"
	TaskManifestHandlingDuration   = taskManifestResponderNamespace + ".Duration"

// TaskStopVeificationACKResponder
	taskStopVerificationACKResponderNamespace = "TaskManifestResponder"
	TaskStoppedMetricName                     = taskStopVerificationACKResponderNamespace + ".TaskStopped"
```

Reply:
Sure, I'll apply this